pkg/tools/gen/genkcl.go

```diff
@@ -0,0 +1,146 @@
+package gen
+
+import (
+	"fmt"
+	"io"
+	"strconv"
+	"strings"
+)
+
+// TODO: is need to parse from Comment. Now get kcl schema from Tag
+type GenKclOptions struct {
+	KclPackage       string
+	ParseFromComment bool
+	ParseFromTag     bool
```

Comment:
Does this field `ParseFromTag` mean that the type and name of KCL will be read from the tag string? Can I directly map from Go Field names and types?

Reply:
Yeah, only will be read from the struct tag current code. Map from go field or field comment will be implemented in future. The next version If the struct field don't have the tag, It will read from go field metadata or go field comment.Current code if the struct field don't have kcl code it will be jump generate.